bot/seasons/evergreen/minesweeper.py

```diff
@@ -0,0 +1,198 @@
+import typing
+from random import random
+
+import discord
+from discord.ext import commands
+
+
+class Minesweeper(commands.Cog):
+    """Play a game of minesweeper."""
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+        self.games: typing.Dict[discord.member, typing.Dict] = {}  # Store the currently running games
+
+    @staticmethod
+    def is_bomb(cell: typing.Union[str, int]) -> int:
+        """Returns 1 if `cell` is a bomb if not 0"""
+        return 1 if cell == "bomb" else 0
```

Comment:
This could probably just be `return cell=="bomb"` as `True` is basically the same as `1`